src/appimagetool.c

```diff
@@ -922,9 +930,9 @@ main (int argc, char *argv[])
             }
         }
         if (verbose)
-            printf("Size of the embedded runtime: %d bytes\n", size);
-        
-        int result = sfs_mksquashfs(source, destination, size);
+            printf("Size of the embedded runtime: %zu bytes\n", size);
+
+        int result = sfs_mksquashfs(source, destination, (int)size);
```

Comment:
The function `sfs_mksquashfs` should probably take a size_t instead of int to avoid the conversion here.